manifest_template.yml

```diff
@@ -12,11 +12,18 @@ APIGEE_ENVIRONMENTS:
     display_name: Internal QA Sandbox
   - name: ref
     display_name: Reference
+    ratelimit: '1000ps'
```

Comment:
Could this be expressed per minute instead? i.e. `60000pm`. This is because of how `ratelimit` buckets `ps` vs `pm` -- using `pm` results in less false positive spike arrests